lib/src/command/global_run.dart

```diff
@@ -13,28 +13,28 @@ import '../utils.dart';
 
 /// Handles the `global run` pub command.
 class GlobalRunCommand extends PubCommand {
-  String get name => "run";
+  String get name => 'run';
   String get description =>
-      "Run an executable from a globally activated package.\n"
+      'Run an executable from a globally activated package.\n'
       "NOTE: We are currently optimizing this command's startup time.";
-  String get invocation => "pub global run <package>:<executable> [args...]";
+  String get invocation => 'pub global run <package>:<executable> [args...]';
   bool get allowTrailingOptions => false;
 
   GlobalRunCommand() {
-    argParser.addFlag("checked",
-        abbr: "c", help: "Enable runtime type checks and assertions.");
-    argParser.addOption("mode", help: "Deprecated option", hide: true);
+    argParser.addFlag('enable-asserts', help: 'Enable assert statements.');
```

Comment:
The following seems clearer / more accurate as help text: **Enable runtime checks of assert statements**. WDYT? /cc @kwalrath @kevmoo

Reply:
I took this directly from `dart --help`. If we want to change this I think we should do it in both places.